src/Slices/Status/UI/Page.test.tsx

```diff
@@ -163,9 +164,13 @@ test("GIVEN StatusPage with support extension WHEN user click download THEN an a
   const downloadButton = screen.getByRole("button", {
     name: "DownloadArchiveButton",
```

Comment:
why is it that some button names are extracted, and others aren't?

Reply:
Github prevents me from commenting on lines that weren't actively part of the changes, so I can't point out the ones you missed. But There are a few still I couldn't comment on...

Reply:
visible text for that button is "Download Archive Button", and only corresponding for value visible above is aria-label

answering second comment: I went through others and for them answer is similar, value asked in parameter "name" is corresponding to aria-label, not visible

Reply:
okay :D